mobile/src/utils/sort.js

```diff
@@ -0,0 +1,43 @@
+const dirMap = {
+  // greater-than
+  gt: { asc: 1, desc: -1 },
+  // less-than
+  lt: { asc: -1, desc: 1 },
+};
+
+const doSort = (A, B, property, direction = 'ASC') => {
+  const a = A[property];
+  const b = B[property];
+
+  if (a < b) {
+    return dirMap.lt[direction.toLowerCase()];
+  }
+  if (a > b) {
+    return dirMap.gt[direction.toLowerCase()];
+  }
+  return 0;
+};
+
+export const createSorter = (...args) => {
+  return (A, B) => {
+    let ret = 0;
+
+    args.some((sorter) => {
+      const { property, direction = 1 } = sorter;
```

Comment:
Should not this be string asc or desc? What if no direction will be in sorter object? Then we will pass direction = 1 to doSort.

Reply:
done